.githooks/pre-commit

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+# This is a simple check to remind people to update the swagger docs
+# if they have made a change to a proto file. It's not iron-clad and non-blocking,
+# but it's better than finding out later in github.
+
+# AMD = added, modified, deleted
+files=$(git diff --cached --name-only --diff-filter=AMD)
+
+LIGHT_RED='\033[1;31m'
+RESET='\033[0m' # No Color
+
+# naive regex check for .proto extension
+if [[ "$files" =~ \.proto ]]
+then
+    # Check if there's a staged diff in /docs
+    result=$(git diff --cached --raw docs/)
```

Comment:
**Request:** Lets check for more than just the docs. The generated Go types are just as important to track.